Update documentation on using the endpoint (#3217)
elastic/docs-content: explore-analyze/elastic-inference/eis.md

serverless: preview
```

> ELSER on EIS enables you to use the ELSER model on GPUs, without having to manage your own ML nodes. We expect significantly better performance for throughput and consistent search latency as compared to ML nodes, and will continue to benchmark, remove limitations and address concerns as we move towards General Availability.

Reviewer: ```suggestion
ELSER on EIS enables you to use the ELSER model on GPUs, without having to manage your own ML nodes. It provides better performance for throughput and more consistent search latency than ML nodes. Elastic will continue to benchmark, remove limitations and address concerns as we move towards General Availability.
```

> ### Usage

Reviewer: ```suggestion
### Usage
```{applies_to}
stack: preview 9.1
serverless: preview
```

```

> You can now use `semantic_text` with the new ELSER endpoint on EIS, see the [instructions to change the inference id](https://www.elastic.co/docs/reference/elasticsearch/mapping-reference/semantic-text#using-elser-on-eis) to use the `.elser-2-elastic` inference endpoint. 

Reviewer: ```suggestion
To learn about using `semantic_text` with the new ELSER endpoint on EIS, refer to [Using ELSER on EIS](https://www.elastic.co/docs/reference/elasticsearch/mapping-reference/semantic-text#using-elser-on-eis). 
```
